### DISCUSSION TITLE
convert pull request comment thread to a discussion

### DISCUSSION THREAD

**Discussion by pathouse** [community/community#17013] [Discussions]
one of the reasons my team recently started looking into using discussions in private repos is that we often have larger architecture/etc. conversations that are spawned from a comment on a specific pull request - the conversation tends to happen in that pull request thread even though it's not directly related to a review of that code and it can be a little distracting

an ideal workflow for us would be to convert that comment thread into a discussion so that we could carry on w/ it in discussions and keep the pull request focused on code review - ideally with a reference back to the specific diff that kicked things off

I understand this may not be a very common use case but just wanted to share

**Comment by lgritz** [comment]
I realize this is an old discussion, but this situation just came up in a project I run as well. It would be very helpful.

**Comment by sargis-tovmasyan** [comment]
Totally agree - this happens to us too. PR comments often spark bigger architecture discussions that don’t really belong in the code review. Being able to move those threads into a Discussion, with a link back to the original diff, would help keep PRs focused and make deeper conversations easier to manage and reference later. Definitely a useful workflow, even if it’s not super common yet.
